diff --git a/node_modules/i18next/index.d.ts b/node_modules/i18next/index.d.ts
index b10306e..e3cb27c 100644
--- a/node_modules/i18next/index.d.ts
+++ b/node_modules/i18next/index.d.ts
@@ -1,4 +1,5 @@
 // Internal Helpers
+// @ts-ignore
 import type { $Dictionary } from './typescript/helpers.d.ts';
 import type {
   DefaultNamespace,
@@ -11,8 +12,10 @@ import type {
   ResourceKey,
   ResourceLanguage,
   TOptions,
+  // @ts-ignore
 } from './typescript/options.d.ts';
 
+// @ts-ignore
 import type { KeyPrefix, TFunction } from './typescript/t.d.ts';
 
 export interface WithT<Ns extends Namespace = DefaultNamespace> {
@@ -494,6 +497,7 @@ export interface i18n {
   emit(eventName: string, ...args: any[]): void;
 }
 
+// @ts-ignore
 export type * from './typescript/options.d.ts';
 export type {
   FallbackLngObjList,
@@ -513,7 +517,9 @@ export type {
   Namespace,
   DefaultNamespace,
   FlatNamespace,
+  // @ts-ignore
 } from './typescript/options.d.ts';
+// @ts-ignore
 export type * from './typescript/t.d.ts';
 export type {
   TFunction,
@@ -521,6 +527,7 @@ export type {
   TFunctionReturn,
   TFunctionDetailedResult,
   KeyPrefix,
+  // @ts-ignore
 } from './typescript/t.d.ts';
 
 declare const i18next: i18n;
